hbase-server/src/main/java/org/apache/hadoop/hbase/master/procedure/DeleteTableProcedure.java

```diff
@@ -93,7 +93,6 @@ protected Flow executeFromState(final MasterProcedureEnv env, DeleteTableState s
         case DELETE_TABLE_PRE_OPERATION:
           // Verify if we can delete the table
           boolean deletable = prepareDelete(env);
-          releaseSyncLatch();
```

Comment:
Why this change? IIRC, this is for keep the old behavior for old client, where we will return immediately after submitting the delete table procedure.